### PR TITLE
Adds Heroku To Aws Migration guide

### PR DESCRIPTION
**Pull request by plandino**

### CODE REVIEW

File: infraestructure/docs/heroku-to-aws-migration.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Heroku To AWS Migration Guide
+
+When migrating an app from Heroku to AWS, you should take into account the following steps:
+
+1. Uncomment the *.ebextensions* files. Delete the *.01_crong.config* file if you don't schedule jobs with whenever. ([Sidekiq Scheduler gem](https://github.com/moove-it/sidekiq-scheduler) is recommended, but if you already started using whenever just uncomment the lines in this file an keep it).
```
Comment (mdesanti) [inline infraestructure/docs/heroku-to-aws-migration.md]: Shouldn't `*.01_crong.config*` be `*01_cron.config*`?

File: infraestructure/docs/heroku-to-aws-migration.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+Add this line in the *config/database.yml* file
+
+```
+production:
+  ...
+  schema_search_path: 'desired_schema'
+```
+
+3. Copy the global variables from Heroku to AWS
```
Comment (mdesanti) [inline infraestructure/docs/heroku-to-aws-migration.md]: We could have a script that does this. Not for this PR, but later on.

Comment (plandino) [inline infraestructure/docs/heroku-to-aws-migration.md]: Good idea, it's really annoying to copy all the variables

**Comment by mdesanti** [comment]
@plandino can you fix merge conflicts so we can merge it?

**Pull request merged**